porch/config/deploy/6-rbac-bind.yaml

```diff
@@ -21,6 +21,19 @@ roleRef:
   kind: ClusterRole
   name: aggregated-apiserver-clusterrole
 subjects:
-- kind: ServiceAccount
-  name: porch-server
-  namespace: porch-system
+  - kind: ServiceAccount
+    name: porch-server
+    namespace: porch-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding # TODO: fix permission
```

Comment:
Did I do this?  Can we just bind it to the functions namespace?

Reply:
Updated.